api/app/Http/Controllers/BaseController.php

```diff
@@ -153,68 +157,4 @@ public function deleteMany(Request $request)
 
         return response(null, 204);
     }
```

Comment:
Just a general question about using the repository pattern, how will we use it to save foreign entities? I really like the simple methods that Eloquent provides and I hope that repository pattern doesn't conflict with this?